apps/__main__.py

```diff
@@ -0,0 +1,39 @@
+'''The morph-tool command line launcher'''
+import logging
+
+import click
+from neurom import load_neuron
+
+logging.basicConfig()
+logger = logging.getLogger('morph_tool')
+logger.setLevel(logging.INFO)
+
+
+@click.group()
+def cli():
+    '''The CLI entry point'''
+
+
+@cli.command(short_help='A simple neuron viewer')
+@click.argument('input_file')
+@click.option('--plane', type=click.Choice(['3d', 'xy', 'yx', 'yz', 'zy', 'xz', 'zx']),
+              default='3d')
+@click.option('--backend', type=click.Choice(['plotly', 'matplotlib']),
+              default='plotly')
```

Comment:
should this really be the default when plotly is an extra?

Reply:
Changed to matplotlib